src/output-json-alert.c

```diff
@@ -230,8 +232,22 @@ static void AlertJsonSourceTarget(const Packet *p, const PacketAlert *pa,
     json_object_set_new(ajs, "target", tjs);
 }
 
+static void AlertJsonMetadata(const PacketAlert *pa, json_t *ajs)
+{
+    if (pa->s->metadata) {
+        DetectMetadata* kv = pa->s->metadata;
+        json_t *mjs = json_object();
+        while (kv) {
```

Comment:
I think this should create a json array instead. E.g. sid 2003380 has
<pre>
metadata:affected_product Any, attack_target Client_Endpoint, deployment Perimeter, tag User_Agent, tag Trojan_Downloader, signature_severity Major, created_at 2010_07_30, updated_at 2017_10_30;
</pre>
Note the multiple 'tag' occurrence.

Alert portion:
<pre>
{"updated_at":"2017_10_30","created_at":"2010_07_30","signature_severity":"Major","tag":"User_Agent","deployment":"Perimeter","attack_target":"Client_Endpoint","affected_product":"Any","former_category":"TROJAN"}
</pre>
On the first tag was added.